packages/cli/src/commands/create/release.ts

```diff
@@ -83,6 +85,21 @@ export const createReleaseCommand = async ({
 
   const { release, app, publisher } = await getConfigWithChecks(buildToolsPath);
 
+  const apkEntry = release.files.find((asset: any) => asset.purpose === "install")!;
+  const size = (await fs.promises.stat(apkEntry.uri)).size;
+  if (size > 100 * 1024 * 1024) {
+    showMessage(
+      "APK over 100MB!!",
+      "Your APK file is over 100MB, which may result in requiring WiFi or other non-metered download operations for end users.\n\n" +
+      "dApp store currently supports only `arm64-v8a` architecture.\n" +
```

Comment:
Again, this is too much information for the CLI. Let's keep it to my original suggestion in the first review.

If we want to provide best practices information, please add it to the publishing docs here in this repo. That's where it should be.

Reply:
Isn't it better to warn the developers before hand rather than at review time that their app is not download friendly. 

Architecture specific comment is not very obvious to the developers since most are used to Play Store separating the binaries for them. 

One time warning here vs repeated efforts while reviewing the app and then pointing the publishers to the docs would be wasting everyone's time

Reply:
Well the thing is, we're not going to be rejecting apps based on size, at least not at first. Also, many of these developers won't have ever released to play store.

Basically, we have to minimize the assumptions we make about developers/publishers at this point. There is a time and a place for this kind of thing, I just don't think it's this way, here in the CLI tooling, right now.

@sdlaver thoughts?

Reply:
This isn't a rejection message, it's a simple warning which mentions `Ignore recommendations if not applicable`

this is to set expectations that their download/update performance could be impacted due to a large app size.

Reply:
Right. Hence my original comments about the general warning. It seems reasonable to have general size constraint warning just so they know. But having a big chunk of information isn't want the CLI is for - that's what our docs are for.

But let's get a 3rd opinion here. I think we need to consider these sorts of things in the context of where the tooling is going.

Reply:
IMO, we need to resolve 2 things before we will be able to reach consensus on this change
1. What are the existing requirements enforced by the Solana dApp Store client for large APKs? Is there a change in behavior above a certain size?
2. What guidance is appropriate to give? I'd posit that the blurb in this PR is too prescriptive. I'd propose that the content of the warning should include a statement of impact ("large download, may require Wi-Fi", etc) plus where to go to find out more (a link to our docs on GitHub, where we can provide much more substantial advice on how to address it.

I'd start by filing an issue to get #2 discussed and added to the docs. Once we have that, we can revisit a PR to add the warning.

Reply:
Yeah sounds good. Close for now, and take the discussion to an issue!